composer.lock

```diff
@@ -2046,6 +2044,52 @@
         }
     ],
     "packages-dev": [
+        {
+            "name": "digitalbazaar/json-ld",
```

Comment:
There shouldn't be a reference to `digitalbazaar/json-ld` in the `composer.lock` anymore, and especially not in `packages-dev`. We'll have to check the `composer.json` again.

Reply:
Then composer.json doesn't contain "digitalbazaar" anymore.

Reply:
Do I have to manually remove something (by the use of some command?)

Reply:
I don't think so, I will be able to rebuild the `composer.lock` from scratch once this PR is merged. This isn't that big of a deal, just odd.

Reply:
Okay.